base.yaml

```diff
@@ -1,24 +1,24 @@
 kubeContext: default
 server_name: example.com
-maintainer_email: MAINTAINER_EMAIL
+maintainer_email: MAINTAINER_EMAIL@example.com
 kafka_num_brokers: 3
-postgres_password: password
 
 # --------------------------------------------------------- 00-init.yaml ---------------------------------------------------------
 
-mongodb:
+graylog:
```

Comment:
MongoDB and Elasticsearch are dependencies of Graylog, they should be running before Graylog